src/modules/navigator/mission_block.cpp

```diff
@@ -278,9 +278,7 @@ MissionBlock::is_mission_item_reached()
 			float acceptance_radius = _navigator->get_acceptance_radius();
 
 			// We use the acceptance radius of the mission item if it has been set (not NAN)
-			// but only for multicopter.
-			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING
-			    && PX4_ISFINITE(_mission_item.acceptance_radius) && _mission_item.acceptance_radius > FLT_EPSILON) {
+			if (PX4_ISFINITE(_mission_item.acceptance_radius) && _mission_item.acceptance_radius > FLT_EPSILON) {
```

Comment:
It looks dangerous to me to always overwrite the acceptance radius calculated by the L1 guidance. What I could live with is taking the max(L1_distance, mission_item.acceptance_raduis). Then one could increase the acceptance radius if needed, but not go below vehicle constraints (which is more or less the L1 distance).

Reply:
Well, it would be upon pilot to validate the required acceptance radius but I agree it makes also sense not to allow him or her to go below the acceptance radius calculated by the L1 guidance.